algorithms/CPlusPlus/Graphs/prim-algorithm.cpp

```diff
@@ -0,0 +1,144 @@
+#include <limits.h>
+
+#include <iostream>
+#include <set>
+#include <vector>
+#include <map>
+
+#define MAXNUMVERTICES 25
+
+using namespace std;
+
+struct Edge
+{
+    int from, to, weight;
+    Edge();
+    Edge(int source, int dest, int wt) : from(source), to(dest), weight(wt) {};
+};
+
+class Graph
+{
+private:
+    int graphMatrix[MAXNUMVERTICES][MAXNUMVERTICES] = { 0 };
+    set<int> vertices;
+    Edge findMinEdge(set<int>& s, set<int>& v_s);
+
+public:
+    Graph() {};
```

Comment:
since you are not overloading the constructor and if you have no use for a constructor, you don't have to create an empty constructor explicitly. During object creation `default constructor`  will be implicitly called.
```suggestion
  
```